### PR TITLE
Articles Starting with W: Rename filenames, add titles

### PR DESCRIPTION
**Pull request by Rafase282**
Refer #889

### CODE REVIEW

File: Wiki-Central.md [hunk 1]
```diff
@@ -2,18 +2,15 @@
 
 Here you will find all wiki related information with regards to guidelines for contributing, moderating, and tips on using the wiki.
 
-## Table of Content
+### Table of Content
```
Comment (theflametrooper) [inline Wiki-Central.md]: ~~Table of Content~~ Table of Contents

**Comment by theflametrooper** [comment]
### **LGTM :+1:**

File: Wiki-Contribute-Online.md [hunk 2]
```diff
@@ -52,23 +63,28 @@ Always delete the branch after the pull request has been merged or closed. There
...
 If you want to use a fork instead of a branch then you will click the link and get something that looks like the image from _Figure 2c_ bellow.
+
 - _Figure 2c_ ![Fork PR](http://i.imgur.com/0PmVLDL.png)
 
 Going back to _Figure 2b_ you will note that you get information like the base branch, the branch with your changes, and how many commits there are along with file changes. From there it's the same as the tutorial for uploading files, please see _Figures 1b_ and onwards with the details that come along with it. But basically you just fill out the details and submit the pull request.
 
 However, something new is the fact that you can **Assign** people, usually contributors with the **Assignee** Option on the far right side of _Figure 2b_. Above it, you will select the milestone for the pull request if any, and above that, select the labels that would fit your pull request. These are all optional though. If you want to learn more about labels then check [this article.](Select-Issues-for-Contributing-Using-Labels)
 
-### 3. New File Button
+### 3\. New File Button
```
Comment (abhisekp) [inline Wiki-Contribute-Online.md]: Why the `\ ` before the dot `.`?

**Comment by abhisekp** [comment]
@Rafase282 **File Rename:-** (_typo_)

```diff
- FreeCodeCamp-Adventages.md
+ FreeCodeCamp-Advantages.md
```

**Pull request merged**